src/crypto/ocb_internal.cc

```diff
@@ -435,11 +459,11 @@ static inline void AES_encrypt(unsigned char *src, unsigned char *dst, AES_KEY *
 }
 #if 0
 /* unused */
-static inline void AES_decrypt(unsigned char *src, unsigned char *dst, AES_KEY *key) {
-	AES_encrypt(src, dst, key);
+static inline void decrypt(unsigned char *src, unsigned char *dst, KEY *key) {
+	encrypt(src, dst, key);
 }
 #endif
-static inline void AES_ecb_encrypt_blks(block *blks, unsigned nblks, AES_KEY *key) {
+static inline void ecb_encrypt_blks(block *blks, unsigned nblks, KEY *key) {
```

Comment:
static inline is redundant in a .cc file. Should we just go down to static?

Reply:
Sure, done.